File: A1/A1-002/A1-002.cpp
#include<iostream>
using namespace std;

int money;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input amount of money
    cin >> money;
    // Output amount of coins;
    cout << "10 = " << money / 10 << '\n';
    money %= 10;
    cout << "5 = " << money / 5 << '\n';
    money %= 5;
    cout << "2 = " << money / 2 << '\n';
    money %= 2;
    cout << "1 = " << money << '\n';
    return 0;
}

File: A1/A1-002/A1-002.py
# --------------------------------------------------
# File Name : A1-002.py
# Problem   : Money Exchange
# Author    : Worralop Srichainont
# Date      : 2026-01-01
# --------------------------------------------------

# Input amount of money
money = int(input().strip())

# Output amount of coins
print(f"10 = {money // 10}")

money %= 10
print(f"5 = {money // 5}")

money %= 5
print(f"2 = {money // 2}")

money %= 2
print(f"1 = {money}")
